cmd/tx/register/bash_commands.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func getRegisterRollappCmd(rollappConfig config.RollappConfig) *exec.Cmd {
 	cmdArgs := []string{
-		"tx", "rollapp", "create-rollapp", rollappConfig.RollappID, "stamp1", "genesis-path/1", "3", "3", `{"Addresses":[]}`,
+		"tx", "rollapp", "create-rollapp", rollappConfig.RollappID, "3", `{"Addresses":[]}`,
```

Comment:
we need to supply the `tokenMetada`, so it will be used by the denommetada middleware
I had a branch for it IIRC